src/App.cpp

```diff
@@ -40,7 +40,7 @@ void App::Update() {
     }
 }
 
-void App::End() {
+void App::End() const {
     Core::Context::GetInstance()->SetExit(true);
```

Comment:
Why can the setter be const?

Reply:
`Context::SetExit()` itself is not a const method.

Reply:
`Context::SetExit()` itself is not a const method.